### PR TITLE
Hammer mods

### PR DESCRIPTION
**Pull request by Dentist2904**
## About The Pull Request
Adds a tool mod with hamming and a one star hammer

## Details

<details>
  <summary>adds a tool mod with hamming and a one star hammer</summary>

</details>

## Screenshots

<details>
  <summary>Expand</summary>

one_star_hammer

</details>

### CODE REVIEW

**Comment by Dentist2904** [comment]
this is done if no one has anything to change

**Comment by TorinoFermic** [comment]
Right now, this tool does not have a cell check to prevent a runtime

File: code/game/objects/items/weapons/tools/hammer.dm [hunk 1]
```diff
@@ -17,11 +17,45 @@
...
+	toggleable = TRUE
 	degradation = 0.7
-	use_power_cost = 0.22
-	suitable_cell = /obj/item/weapon/cell/small
\ No newline at end of file
+	use_power_cost = 3
+	suitable_cell = /obj/item/weapon/cell/medium
+	max_upgrades = 1
+
+/obj/item/weapon/tool/hammer/powered_hammer/turn_on(mob/user)
```
Comment (MadmanMartian) [inline code/game/objects/items/weapons/tools/hammer.dm]: Needs a check to see if it actually has a cell

File: code/game/objects/items/weapons/tools/hammer.dm [hunk 2]
```diff
@@ -17,11 +17,47 @@
...
+	if(!cell)
+		return 0
+	if(cell.charge > use_power_cost)
+		to_chat(user, SPAN_NOTICE("You switch [src] on."))
+		..()
+	else
+		item_state = initial(item_state)
+		to_chat(user, SPAN_WARNING("[src] does not have a power source installed."))
```
Comment (TorinoFermic) [inline code/game/objects/items/weapons/tools/hammer.dm]:
```suggestion
		to_chat(user, SPAN_WARNING("[src] seems have a dead cell."))
```

Comment (TorinoFermic) [reply code/game/objects/items/weapons/tools/hammer.dm]: Last suggestion and that's ready to be merged.

**Pull request merged**